src/assets/wise5/directives/teacher-summary-display/dialog-guidance-teacher-summary-display.component.html

```diff
@@ -0,0 +1,36 @@
+<mat-card appearance="outlined" class="summary-card">
+  <mat-card-content>
+    @if (hasWarning) {
+      <p class="warn center">{{ warningMessage }}</p>
+    }
+    @if (doRender) {
+      <div class="wrapper">
+        <div class="idea-group" id="most-common-ideas">
+          <h2>Most Common Ideas Detected:</h2>
```

Comment:
Add i18n tag to this and other text elements on this page that are translatable?
```suggestion
          <h2 i18n>Most Common Ideas Detected:</h2>
```